gui/dialog/puck_dialog.py

```diff
@@ -0,0 +1,94 @@
+import logging
+import typing
+from typing import Optional
```

Comment:
is there any reason this import needs to be here? seems like the Optional item is never used

Reply:
Removed those imports in https://github.com/NSLS-II/lsdc/pull/286/commits/61f92ac3c9f28c816fcf64848f1240f421b9a8a0